docs/source/getting_started/woodwork_types.ipynb

```diff
@@ -0,0 +1,386 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b95b28c1",
+   "metadata": {},
+   "source": [
+    "# Woodwork Typing in Featuretools\n",
+    "\n",
+    "Featuretools relies on having consistent typing across the creation of EntitySets, Primitives, Features, and feature matrices. Previously, Featuretools used its own type system that contained objects called Variables. Now and moving forward, Featuretools will use an external data typing library for its typing: [Woodwork](https://woodwork.alteryx.com/en/stable/index.html).\n",
+    "\n",
+    "Understanding the Woodwork types that exist and how Featuretools uses Woodwork's type system will allow users to build EntitySets that best represent their data, understand the possible input and return types for Featuretools' Primitives, and understand what features will get generated from a given set of data and primitives.\n",
+    "\n",
+    "Read the [Understanding Woodwork Logical Types and Semantic Tags](https://woodwork.alteryx.com/en/stable/guides/logical_types_and_semantic_tags.html) guide for an in-depth walkthrough of the available Woodwork types that are outlined below.\n",
+    "\n",
+    "For users that are familiar with the old `Variable` objects, the [Transitioning from Variables to Woodwork](link) guide will be useful for converting Variable types to Woodwork types.\n",
+    "\n",
+    "## Physical Types \n",
+    "Physcial types define how the data in a Woodwork DataFrame is stored on disk or in memory. You might also see the physical type for a column referred to as the column’s `dtype`.\n",
+    "\n",
+    "Pandas, Dask, and Koalas DataFrames rely on these dtypes when performing DataFrame operations, so knowing a Woodwork DataFrame's physical types is important for any DataFrame operations that might be performed. Each Woodwork `LogicalType` class has a single physical type associated with it.\n",
```

Comment:
I don't think this comes off that clear. I'd suggest something like:

>  Knowing a Woodwork DataFrame's physical types is important because 
Pandas, Dask, and Koalas rely on these types when performing DataFrame operations.

Reply:
yep, your version is much nicer--will change!